politeiawww/cmd/politeiawwwcli/commands/edituser.go

```diff
@@ -32,17 +37,60 @@ Response:
 {}`
 
 type EditUserCmd struct {
-	EmailNotifications *uint64 `long:"emailnotifications" optional:"true" description:"Whether to notify via emails"`
+	Args struct {
+		NotifType string `long:"emailnotifications" description:"Email notifications"`
+	}
 }
 
 func (cmd *EditUserCmd) Execute(args []string) error {
+
+	EmailNotifs := map[string]v1.EmailNotificationT{
```

Comment:
This email notifications list should reference the actual notification variables defined in the api instead of hard coding in the values.

Example:
```
"userproposalchange": v1.NotificationEmailMyProposalStatusChange,
```

Also, I would avoid capitalizing local variables like `EmailNotifs`.  Exported variables in go are capitalized, so capitalizing a non-exported variable might be seen as confusing.  `emailNotifs` would be better.

Reply:
Done. FYI, we also hard code values in `manageuser.go` (the function I was modeling this on). Should I also update `manageuser` to reference variables in v1? If so, do it in this PR, or open another?

Reply:
You can include it in this PR.